zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -207,6 +207,7 @@ where
         let from = extfvk
             .fvk
             .vk
+            .ivk()
```

Comment:
Should we add a convenience `to_payment_address` method so that it's possible to skip the `.ivk()` if you don't need it? I'm thinking partly of compatibility with third-party code already using these APIs.